connected-car-lagom-impl/src/main/resources/application.conf

```diff
@@ -23,75 +23,60 @@ akka.actor.serialization-bindings {
 }
 
 akka {
-
   loglevel = DEBUG
 
-  discovery {
-    method = marathon-api
-
-    marathon-api {
-      class = akka.discovery.marathon.MarathonApiSimpleServiceDiscovery
-
-      # URL for getting list of apps from Marathon. Verified on OSS DC/OS 1.8, 1.9
-      app-api-url = "http://marathon.mesos:8080/v2/apps"
-
-      # The name of the akka management port - this cannot have underscores or dashes (env var name)
-      app-port-name = "akkamgmthttp"
-
-      # Used to find other apps running by Marathon. This will be passed as the label query string parameter
-      # to the apps-api-url defined above.
-      # `%s` will be replaced with the configured effective name, which defaults to the actor system name
-      app-label-query = "ACTOR_SYSTEM_NAME==%s"
-    }
-  }
-
   actor {
     provider = "cluster"
   }
 
-  remote {
-    netty.tcp {
-      hostname = "localhost"
-      hostname = ${?HOST}
-      port = ${?PORT_2551}
+  discovery {
+    method = marathon-api
+  }
 
-      bind-hostname = 0.0.0.0
-      bind-port = 2551
+  io {
+    dns {
+      resolver = async-dns
+      async-dns {
+        provider-object = "com.lightbend.rp.asyncdns.AsyncDnsProvider"
+        resolve-srv = true
+        resolv-conf = on
+      }
```

Comment:
This `io` section uses the service locator from the `reactive-lib` project. In the future, we hope to move that over to `akka-management`.